Packs/NetQuestOMX/Integrations/NetQuestOMX/NetQuestOMX.py

```diff
@@ -0,0 +1,489 @@
+                method="GET",
+                url_suffix=f"/api/Systems/Slot/{slot_number}/Ipfix/Status/Metering",
+                ok_codes=(200,)
+            )
+        except Exception as e:
+            raise DemistoException(
+                "An error was occurred when requesting for an event of Metering Stats type."
+            ) from e
+
+        metering_stats_event["STAT_TYPE"] = 'MeteringStats'
+
+        return metering_stats_event
+
+    def export_stats_request(self, slot_number: str, port_number: str) -> dict[str, Any]:
+        try:
+            export_stats_event = self._http_request(
+                method="GET",
+                url_suffix=f"/api/Systems/Slot/{slot_number}/Ipfix/Status/Export",
+                ok_codes=(200,)
+            )
+        except Exception as e:
+            raise DemistoException(
+                "An error was occurred when requesting for an event of Export Stats type."
+            ) from e
+
+        export_stats_event["STAT_TYPE"] = 'ExportStats'
+
+        return export_stats_event
+
+    def export_peaks_FPS_request(self, slot_number: str, port_number: str) -> dict[str, Any]:
+        try:
+            export_peaks_FPS_event = self._http_request(
+                method="GET",
+                url_suffix=f"/api/Systems/Slot/{slot_number}/Ipfix/Status/ExportHwm",
+                ok_codes=(200,)
+            )
+        except Exception as e:
+            raise DemistoException(
+                "An error was occurred when requesting for an event of Export Peaks FPS type."
+            ) from e
+
+        export_peaks_FPS_event["STAT_TYPE"] = 'ExportPeaksFPS'
+
+        return export_peaks_FPS_event
+
+    def optimization_stats_request(self, slot_number: str, port_number: str) -> dict[str, Any]:
+        try:
+            optimization_stats_event = self._http_request(
+                method="GET",
+                url_suffix=f"/api/Systems/Slot/{slot_number}/Port/{port_number}/EthernetInterfaces/Status/EthRxTx",
+                ok_codes=(200,)
+            )
+        except Exception as e:
+            raise DemistoException(
+                "An error was occurred when requesting for an event of Optimization Stats type."
+            ) from e
+
+        optimization_stats_event["STAT_TYPE"] = 'OptimizationStats'
+
+        return optimization_stats_event
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def address_list_upload_command(client: Client, args: dict):
+    """
+    This function uploads a .txt file with address list to the appliance.
+    The appliance temporarily stores the file until it is saved to the Library and replaces any previously loaded list file.
+
+    Returns:
+        A CommandResults containing a success indication or a DemistoException.
+    """
+    entry_id = args["entry_id"]  # a required argument
+    file_info = demisto.getFilePath(entry_id)
+    file_path = file_info['path']
+    file_name = file_info['name']
+
+    shutil.copy(file_path, file_name)
```

Comment:
use `with tempfile.NamedTemporaryFile` (to create a temporary file that will delete after use, instead of copying-then-deleting